tests/integration/utils2.go

```diff
@@ -781,6 +766,10 @@ func refreshCollections(
 ) {
 	s.collections = make([][]client.Collection, len(s.nodes))
 
+	for i := len(s.nodePeerCollections); i < len(s.collectionNames); i++ {
+		s.nodePeerCollections = append(s.nodePeerCollections, make(map[int]struct{}))
```

Comment:
todo: This looks like it will break when collections are removed/deactivated?  Users currently can do this, although we likely have no P2P tests for it atm.

There may be other test things that currently break when doing that, but we should avoid adding new ones.

Reply:
This should be easy to handle. Just need to find which collections to deactivate from the p2p state.

- [ ] handle patchCollection